qemu/tests/live_snapshot_simple.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from virttest import error_context
+from qemu.tests import live_snapshot_basic
+
+
+class LiveSnapshotSimple(live_snapshot_basic.LiveSnapshot):
```

Comment:
Actually no.
I have added notes about it in the docstring.
Just want to keep the same formation like drive_mirror and block_stream.

 + """
+ This class is not extending any func() from its parent,
+ just for keeping same formation as other similar classes.
+ """